index.html

```diff
@@ -17,29 +17,37 @@
 <body>
     <header>
         <nav class="navbar">
+          <div class="logo" href="#">Maryam</div>
           <ul class="navbar-flex">
-            <li class="nav-logo"><a href="#">Maryam</a></li>
-            <li class="nav-logo">
+            <li class="nav-link"><a href="#">Portfolio</a></li>
+            <li class="nav-link"><a href="#">About</a></li>
+            <li class="nav-link"><a href="#">Contact</a></li>
+            <li class="envelope"><a href="#"></a></li>
+            <li class="mob-bars">
               <a href="#"><i class="fa-solid fa-bars"></i></a>
             </li>
```

Comment:
- [ ] The hamburger button is not displayed in the mobile version, kindly make sure to fix that issue.